.github/workflows/release-pr.yml

```diff
@@ -174,8 +174,9 @@ jobs:
 
           EOF
       - name: Upload PR Event as Artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: preview-spec
+          name: preview-spec-event
           path: event.json
           retention-days: 2
+          overwrite: true
```

Comment:
This follows the `v3` behaviour by overwriting the `event.json` if the PR is closed.

https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md#overwriting-an-artifact